src/zcl_abapgit_file_status.clas.abap

```diff
@@ -13,46 +13,59 @@ CLASS zcl_abapgit_file_status DEFINITION
 
   PRIVATE SECTION.
 
-    CLASS-METHODS:
```

Comment:
The following section is just formatting done by the pretty printer. No functional changes.